operator/Makefile

```diff
@@ -7,6 +7,14 @@
 # dropped (e.g. 3.0.61.1 -> 3.61.1) and development version ".x" is changed to ".0" (e.g. 3.0.61.x-123 -> 3.0.61.0-123).
 VERSION ?= $(shell $(MAKE) --quiet --no-print-directory -C .. tag | sed -E 's@^(([[:digit:]]+\.)+)x(-)?@\10\3@g' | sed -E 's@^3.0.([[:digit:]]+\.[[:digit:]]+)(-)?@3.\1\2@g')
 
+# The operator version used by ClusterServiceVersion object and related tooling (bundle, index). It is prefixed with "v".
+CSV_VERSION ?= v$(VERSION)
+# Operator index image tag to install from, used by deploy*-via-olm targets.
+INDEX_IMG_TAG ?= $(CSV_VERSION)
+# Operator version to install and channel to install from, used by deploy-(dirty-tag-)via-olm targets.
+INSTALL_VERSION ?= $(CSV_VERSION)
+INSTALL_CHANNEL ?= latest
```

Comment:
My suggestions should allow not introducing `INSTALL_VERSION` and `INSTALL_CHANNEL` variables here which in my opinion make things more confusing.

Also, I think `CSV_VERSION` can be deleted and `INDEX_IMG_TAG` be kept as the only thing. Perhaps as

```
# Tag of operator bundle and index images is version prefixed with "v".
INDEX_BUNDLE_IMG_TAG ?= v$(VERSION)
```

Reply:
I'd like to keep `CSV_VERSION` as a simple way to specify both the tag and the version to install in one go.